.bazelrc

```diff
@@ -21,6 +21,8 @@ build --features=swift.opt_uses_osize
 build --features=swift.opt_uses_wmo
 build --features=swift.remap_xcode_path
 build --features=swift.use_global_module_cache
+build --features=swift.emit_swiftinterface
```

Comment:
This option is currently making the flatbuffer dep fail, not sure what we need this for